util/customTransformer-v1.js

```diff
@@ -2,8 +2,18 @@ const ivm = require("isolated-vm");
 const stats = require("./stats");
 
 const { getPool } = require("./ivmPool");
+const { getFactory } = require("./ivmFactory");
+const logger = require("../logger");
 
-
+/*
+Transform VM aquire mode is
+ Pooled if env variable ON_DEMAND_ISOLATE_VM = 0
```

Comment:
Can you make the env variable boolean instead of 1 and 0?

Reply:
Done.